pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -156,7 +156,7 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 		AllowedCIDRs: allowedCIDRs,
 		Tags:         lb.Tags,
 	}
-	return nil
+	return true, nil
```

Comment:
Why return `true` here? It seems conflicting to me: no error but terminal error :thinking:

Reply:
ok, make it false :)
technically we return err=nil so true/false doesn't matter :)